plugin.xml

```diff
@@ -3,5 +3,5 @@
   <feature extension="package.version" value="2.0.0"/>
   <transtype name="org.dita-ot.html"/>
   <feature extension="dita.conductor.target.relative" file="integrator.xml"/>
-  <feature extension="dita.conductor.html5.param" file="params.xml"/>
+  <feature extension="dita.conductor.html.param" file="params.xml"/>
```

Comment:
Changing this will on the other hand break 2.3 code and since this plugin is only used by the DITA-OT project itself internally, I'm not convinced this should be changed.